lib/host.rb

```diff
@@ -854,20 +854,36 @@ def exec_raw(*commands, **opts)
         commands = ["chroot", "/host/", "bash", "-c", commands_to_string(commands)]
       end
 
+      unless node.env.opts[:host_debug_image] == "support-tools"
+        @result = node.env.admin.cli_exec(:get, n:"openshift-multus", resource: "pod", l: "app=multus", output: 'jsonpath={.items[0].spec.containers[?(@.name=="kube-multus")].image}')
+        if @result[:success]
+          # we could cache this but then we would have to re-get after upgrade
+          opts[:image] = @result[:response].strip
+        end
+      end
+
+      exec_opts = {}
+      # override image, arg order matters, image needs to go before --
+      if opts[:image]
+        exec_opts[:image] = opts[:image]
```

Comment:
@rbbratta could be replaced with `exec_opts[:image] ||= opts[:image]`

Reply:
very cool, I was not aware of `||=`, but I don't think it matches in this case.  `a ||=b`  seems to roughly approximate `a || a = b`, but we want `b || a = b`

```
opts[:image] || exec_opts[:image] = opts[:image]
```

Had to read this to figure it out:
http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html